pkg/virt-controller/watch/volume-migration/volume-migration.go

```diff
@@ -141,6 +146,34 @@ func ValidateVolumes(vmi *virtv1.VirtualMachineInstance, vm *virtv1.VirtualMachi
 			valid = false
 			continue
 		}
+
+		// DataVolumes with a no-csi storage class
+		if v.VolumeSource.DataVolume != nil {
+			dv, err := storagetypes.GetDataVolumeFromCache(vm.Namespace, v.VolumeSource.DataVolume.Name, dvStore)
+			if err != nil || dv == nil {
+				continue
+			}
+			boolUsePopulator := true
```

Comment:
Quick check here to see if the DataVolume is in `Succeeded` phase. If so, it should be safe to use, can skip the rest of the check